repos.d/gentoo/liguros.yaml

```diff
@@ -1,40 +1,38 @@
 ###########################################################################
 # Liguros
 ###########################################################################
-- name: liguros_20_7
+- name: liguros_21_7
   type: repository
-  desc: LiGurOS 20.7
+  desc: LiGurOS 21.7
   statsgroup: Gentoo
   family: gentoo
   ruleset: [gentoo, liguros]
   color: '1de9b6'
   minpackages: 28000
+  valid_till: 2022-02-01
   default_maintainer: fallback-mnt-liguros@repology
   sources:
     - name:
-        - core-kit
-        - python-kit
-        - python-modules-kit
-        - nokit
+        - liguros-repo
       fetcher: GitFetcher
       parser: GentooGitParser
-      require_md5cache_metadata: true
+      require_md5cache_metadata: false
```

Comment:
Repositories without metadata are not acceptable.

Reply:
https://github.com/repology/repology-updater/blob/master/repos.d/gentoo/sabayon.yaml#L15-L16
https://github.com/repology/repology-updater/blob/master/repos.d/gentoo/calculate.yaml#L15-L16

both repos have that set to false.

Where is the problem then to disable that for liguros repo? Please explain.

Reply:
Other repositories are not related to this case. Metadata is required, change which drops it is a pessimization and won't be accepted.